modules/decline/src/core/OptsVisitor.scala

```diff
@@ -35,19 +42,23 @@ import java.util.UUID
 import smithy4s.schema.CollectionTag
 import smithy4s.schema.CollectionTag.ListTag
 
-object OptsVisitor
-    extends SchemaVisitor[Opts] { self =>
+object OptsVisitor extends SchemaVisitor[Opts] { self =>
 
   private def field[P: Argument](
       hints: Hints
   ): Opts[P] = {
     val fieldName = FieldName.require(hints)
     val isNested = IsNested.orFalse(hints)
-    val doc = hints.get(Documentation).fold("")(_.value)
+    val doc = hints.get(Documentation).map(_.value).toList ::: hints
+      .get(ExternalDocumentation)
+      .toList
+      .flatMap(_.value.map { case (description, link) =>
+        s"\n\n$description\n\n[See more]($link)"
+      }.toList)
```

Comment:
**thought:** rendering markdown here sounds weird, given the output will end up in a terminal - I don't think many (if any) of these read markdown

Reply:
lol copilot generated it .
will remove markdown syntax